utility_analysis_new/dp_engine.py

```diff
@@ -54,21 +54,29 @@ def _create_compound_combiner(
         mechanism_type = aggregate_params.noise_kind.convert_to_mechanism_type()
         budget = self._budget_accountant.request_budget(
             mechanism_type, weight=aggregate_params.budget_weight)
-        return combiners.CompoundCombiner([
-            utility_analysis_combiners.UtilityAnalysisCountCombiner(
-                combiners.CombinerParams(budget, aggregate_params),
-                self._is_public_partitions)
-        ],
+        compound_combiners = []
+        if set(aggregate_params.metrics).issubset({pipeline_dp.Metrics.COUNT}):
```

Comment:
Nit:` if pipeline_dp.Metrics.COUNT in aggregate_params.metrics`

Reply:
Thanks for the catch, changed!